mu/interface.py

```diff
@@ -809,16 +813,55 @@ def process_bytes(self, bs):
         # then move it there.
         while tc.movePosition(QTextCursor.Down):
             pass
-        for b in bs:
-            if b == 8:  # \b
+
+        i = 0
+        while i < len(data):
+            if data[i] == 8:  # \b
                 tc.movePosition(QTextCursor.Left)
                 self.setTextCursor(tc)
-            elif b == 13:  # \r
+            elif data[i] == 13:  # \r
                 pass
+            elif data[i] == 27 and data[i+1] == 91:  # VT100 cursor: <Esc>[
```

Comment:
Presumably; even though it's not specified in PEP8 :-/ 

On a side note, it strikes me that PEP8 errors in travis are detracting from 'real' errors - I get that code style is important, but a 'warning' rather than 'error' approach (if that's possible) would be nicer IMHO. Also, there are occasions where breaking PEP8 makes code more readable (see https://www.python.org/dev/peps/pep-0008/#id15 )